### PR TITLE
Handling fee payment for Community Account

### PR DESCRIPTION
**Pull request by johandroid**
First functionality "hardcoded" in the pallet payments, preparing to move the functionality to the pallet "fee-handler"

### CODE REVIEW

File: runtime/kreivo/src/payments.rs [hunk 1]
```diff
@@ -28,29 +29,35 @@ impl pallet_payments::BenchmarkHelper<AccountId, FungibleAssetLocation, Balance>
...
 		_beneficiary: &AccountId,
 		amount: &Balance,
 		_remark: Option<&[u8]>,
 	) -> Fees<Runtime> {
 		let min = <Assets as fungibles::Inspect<AccountId>>::minimum_balance(*asset);
+
+		// Johan toca usar: try_from_sub_account
+		let is_paying_fees:bool = match PalletId::try_from_sub_account::<CommunityId>(sender) {
+			Some((pallet_id, _)) if pallet_id == crate::communities::CommunityPalletId::get() => true,
+			_ => false
```
Comment (pandres95) [inline runtime/kreivo/src/payments.rs:49]: I guess is the contrary: communities don't pay, while others do.

```suggestion
			Some((pallet_id, _)) if pallet_id == crate::communities::CommunityPalletId::get() => false,
			_ => true
```

File: .gitignore [hunk 2]
```diff
@@ -36,3 +36,4 @@ zombienet-linux
 # IDEs
 .vscode/
 .idea/
+polkadot-sdk
```
Comment (pandres95) [inline .gitignore:39]: Guess it'd be cleaner not to mix repositories in first place. In this case, moving your copy of `polkadot-sdk` out of `virto-node`'s repo seems cleaner than ignoring it.

File: Cargo.toml [hunk 3]
```diff
@@ -52,6 +52,7 @@ pallet-burner = { default-features = false, path = "pallets/burner" }
 pallet-payments = { default-features = false, path = "pallets/payments" }
 pallet-communities = { default-features = false, path = "pallets/communities" }
 pallet-communities-manager = { default-features = false, path = "pallets/communities-manager" }
+pallet-fee-handler = { default-features = false, path = "pallets/fee-handler" }
```
Comment (pandres95) [inline Cargo.toml:55]: Is this a WIP? It'd be nicer to have it as a separate PR. This way, we can immediately approve the _"hardcoded"_ solution after requested changes, and start reviewing the extension pallet later.

**Review by pandres95 — CHANGES_REQUESTED** [review]
Looks nice overall, clever solution for a specific issue. Also, a good start to understanding how to work out on the new pallet. ~If we move that new pallet to another PR, we can leave it for a further review, and still accept the current specific solution that's outlined in the kreivo config.~

File: Cargo.toml (hunk 3 above)
Comment (pandres95) [reply Cargo.toml:55]: Nvm. Noticed this is different from #380

File: runtime/kreivo/src/payments.rs (hunk 1 above)
Comment (olanod) [reply runtime/kreivo/src/payments.rs:49]: This can be caught with proper tests

**Comment by johandroid** [comment]
Wrong behaviour of the proposed code, the branch must be deleted

**Pull request closed without merging**